src/components/CommentForm.jsx

```diff
@@ -0,0 +1,51 @@
+import { useContext, useState } from "react";
+import { UserContext } from "../contexts/User";
+import { postComment } from "../../api";
+function CommentForm({ article_id, setComments }) {
+  const { user } = useContext(UserContext);
+  const [userInfo] = user;
+  const [newComment, setNewComment] = useState("");
+  const [isPosting, setIsPosting] = useState(false);
+
+  function handleChange(event) {
+    setNewComment(event.target.value);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    if (!newComment) {
```

Comment:
I think we could give the client some kind of feedback here to tell them why this failed